src/confluent_kafka/src/Admin.c

```diff
@@ -3351,8 +3351,12 @@ Admin_c_AclBinding_to_py (const rd_kafka_AclBinding_t *c_acl_binding) {
                                 rd_kafka_AclBinding_resource_pattern_type(c_acl_binding));
         cfl_PyDict_SetString(kwargs, "principal",
                                      rd_kafka_AclBinding_principal(c_acl_binding));
-        cfl_PyDict_SetString(kwargs, "host",
-                                     rd_kafka_AclBinding_host(c_acl_binding));
+        if (rd_kafka_AclBinding_host(c_acl_binding)) {
+                cfl_PyDict_SetString(kwargs, "host",
+                                        rd_kafka_AclBinding_host(c_acl_binding));
+        } else {
+                cfl_PyObject_SetItemString(kwargs, "host", Py_None);
+        }
```

Comment:
Please revert this change:

1. This change is not related to `describe_topic`
2. We can't reproduce this
3. It shouldn't be null as per the protocol

Reply:
Not nullable too in DescribeAclsResponse and DeleteAclsResponse

https://github.com/apache/kafka/blob/d65d8867983e54a36178c3a1227b68c8a468d415/clients/src/main/resources/common/message/DescribeAclsResponse.json#L45
https://github.com/apache/kafka/blob/d65d8867983e54a36178c3a1227b68c8a468d415/clients/src/main/resources/common/message/DeleteAclsResponse.json#L49